index.js

```diff
@@ -0,0 +1,513 @@
+		messageEmbed(message, "https://media.giphy.com/media/PKgfwX7ct5f5C/giphy.gif");
+	}
+	if (message.content.toLowerCase().includes('you underestimate my power')) {
+		messageEmbed(message, "https://media.giphy.com/media/FXiLqLk921kOs/giphy.gif"); //@todo This is broken since the link doesn't work anymore, content was removed
+	}
+	if (message.content.toLowerCase().includes('shame on you sweet nerevar')) {
+		message.channel.send("https://www.youtube.com/watch?v=iR-K2rUP86M");
+	}
+
+	// @todo Resolve the below function
+	//THEORY - need test, no idea if this works even remotely - on join apply a role
+	// client.on('guildMemberAdd', guildMember => {
+	// 	// Set the member's roles to a single role
+	// 	if (client.guilds.get(message.guild.id).id == 503178803477086208){ //Private friends discord
+	// 		guildMember.setRoles(['503194198581837844']) //"Clean" role I believe
+	// 		.then(console.log)
+	// 		.catch(console.error);
+	// 	}
+	// });
+
+	//Giphy Search
+	//The reference for emojis only works on my personal discord, unsure how it works on others -----
+	// @todo Its broken at the moment, will need to fix it since it's uploading a file instead of posting gif in chat
+	let smugRef = ["606767823280144385", "606776274551177247", "607150963375079430", "607150995318898699"];
+	if (message.content.startsWith(`${prefix}gif`)){
+		//let member = message.mentions.members.first(); //<= In case want to @ later on
+		giphy.search('gifs', {"q": "smug"}).then ((response) => {
+			let totalResponses = response.data.length;
+			let responseIndex = Math.floor((Math.random() *10) + 1) % totalResponses;
+			let responseFinal = response.data[responseIndex];
+			let smugNum = Math.floor(Math.random() * smugRef.length);
+			message.channel.send("Get smugged <:smug"+smugNum+":"+smugRef[smugNum]+">", { files: [responseFinal.images.fixed_height.url] });
+			console.log(responseFinal.images.fixed_height);
+		}).catch(() => {message.channel.send('Error!');})
+	}
+
+/*
+Spam Command, un-comment with CTRL+/ when want to use the spam command.
+*/
+	// if(message.content.startsWith(`${prefix}spam`)){
+	// 	var input = message.content.split(' ');
+	// 	message.delete();
+	// 	for(i=0;i<input[1];i++){
+	// 		message.channel.send("Ya yeet "+i);
+	// 		if(input[2] == 'd'){
+	// 			//let channel = client.channels.get("7");
+	// 			message.channel.fetchMessages({ limit: 1 }).then(messages => {
+	// 				let lastMessage = messages.first();
+	// 				if (lastMessage.author.bot) {
+	// 				  lastMessage.delete();
+	// 				}
+	// 			}).catch(console.error);
+	// 		}
+	// 	}
+	// }
+
+	// Message Deletion, specify an amount of messages to delete and Cuddles will do so.
+	if(message.content.startsWith(`${prefix}clear`)){
+		clearMessages(message);
+	}
+
+});
+/**
+ *  END CLIENT.ON
+ **/
+
+/**
+ * @pingPong
+ * @param message
+ *
+ * Cuddles will send Pong! back to the user
+ */
+function pingPong(message) {
+	message.channel.send("Pong!");
```

Comment:
might be worth putting all of these functions into separate files and importing them. They blow up your main file.

Reply:
Working on modules at the moment, will update in a later PR